example-get-started/generate.sh

```diff
@@ -194,7 +196,7 @@ dvc exp run --queue --set-param train.min_split=8 --set-param train.n_est=100
 dvc exp run --queue --set-param train.min_split=64 --set-param train.n_est=100
 dvc exp run --run-all -j 2
 # Apply best experiment
-dvc exp apply $(dvc exp show --no-pager --sort-by avg_prec | tail -n 2 | head -n 1 | grep -o 'exp-\w*')
+dvc exp apply $(dvc exp show --sort-by avg_prec --csv | tail -n 1 | cut -d , -f 1)
```

Comment:
Unrelated but this was causing problems with a narrow terminal where exps were shown on multiple lines, so this should be more stable.